pkg/config/deployment.go

```diff
@@ -44,6 +44,8 @@ type GenericDeploymentSpec struct {
 	Timeout Duration `json:"timeout,omitempty" default:"6h"`
 	// List of encrypted secrets and targets that should be decoded before using.
 	Encryption *SecretEncryption `json:"encryption"`
+	// Notification to be sent to users via Slack or email
```

Comment:
nit: `Additional configuration used while sending notification to external services.`

Reply:
26713d1